torch/testing/_internal/inductor_utils.py

```diff
@@ -37,22 +42,60 @@ def test_cpu():
     ):
         return False
 
+
 HAS_CPU = LazyVal(test_cpu)
 
-HAS_TRITON = has_triton()
 
-if HAS_TRITON:
-    import triton
-    TRITON_HAS_CPU = "cpu" in triton.backends.backends
-else:
-    TRITON_HAS_CPU = False
+# Ensure the scheduling backends are registered first.
+init_backend_registration()
```

Comment:
This should be done lazily not at import time since it will prevent people from overriding.